diff -urNp -x '*.orig' cacti-spine-1.2.16.org/spine.conf.dist cacti-spine-1.2.16/spine.conf.dist
--- cacti-spine-1.2.16.org/spine.conf.dist	2020-11-30 19:21:59.000000000 +0100
+++ cacti-spine-1.2.16/spine.conf.dist	2022-02-09 22:10:49.918624593 +0100
@@ -40,11 +40,11 @@
 # | RDB_Pass        The remote database password.                           |
 # +-------------------------------------------------------------------------+
 
-DB_Host       localhost
-DB_Database   cacti
-DB_User       cactiuser
-DB_Pass       cactiuser
-DB_Port       3306
+#DB_Host       localhost
+#DB_Database   cacti
+#DB_User       cactiuser
+#DB_Pass       cactiuser
+#DB_Port       3306
 #DB_UseSSL    0
 #DB_SSL_Key
 #DB_SSL_Cert
diff -urNp -x '*.orig' cacti-spine-1.2.16.org/spine.h cacti-spine-1.2.16/spine.h
--- cacti-spine-1.2.16.org/spine.h	2020-11-30 19:21:59.000000000 +0100
+++ cacti-spine-1.2.16/spine.h	2022-02-09 22:10:49.918624593 +0100
@@ -148,22 +148,19 @@
 #define SPINE_FORK 0
 
 /* locations to search for the config file */
-#define CONFIG_PATHS 4
-#define CONFIG_PATH_1 ""
-#define CONFIG_PATH_2 "/etc/"
-#define CONFIG_PATH_3 "/etc/cacti/"
-#define CONFIG_PATH_4 "../etc/"
+#define CONFIG_PATHS 1
+#define CONFIG_PATH_1 "/etc/webapps/cacti/"
 
 /* config file defaults */
 #define DEFAULT_CONF_FILE "spine.conf"
 #define DEFAULT_THREADS 5
 #define DEFAULT_DB_HOST "localhost"
 #define DEFAULT_DB_DB "cacti"
-#define DEFAULT_DB_USER "cactiuser"
-#define DEFAULT_DB_PASS "cactiuser"
+#define DEFAULT_DB_USER "mysql"
+#define DEFAULT_DB_PASS ""
 #define DEFAULT_DB_PORT 3306
 #define DEFAULT_DB_PREG 0
-#define DEFAULT_LOGFILE "/var/www/html/cacti/log/cacti.log"
+#define DEFAULT_LOGFILE "/var/log/cacti/cacti.log"
 #define DEFAULT_TIMEOUT 294000000
 
 /* threads constants */
diff -urNp -x '*.orig' cacti-spine-1.2.16.org/util.c cacti-spine-1.2.16/util.c
--- cacti-spine-1.2.16.org/util.c	2020-11-30 19:21:59.000000000 +0100
+++ cacti-spine-1.2.16/util.c	2022-02-09 22:10:49.918624593 +0100
@@ -957,9 +957,6 @@ void config_defaults() {
 	STRNCOPY(set.rdb_pass, DEFAULT_DB_PASS);
 
 	STRNCOPY(config_paths[0], CONFIG_PATH_1);
-	STRNCOPY(config_paths[1], CONFIG_PATH_2);
-	STRNCOPY(config_paths[2], CONFIG_PATH_3);
-	STRNCOPY(config_paths[3], CONFIG_PATH_4);
 
 	set.log_destination = LOGDEST_FILE;
 }
