--- cacti-spine-0.8.7g/spine.h~	2010-07-12 12:40:01.000000000 +0300
+++ cacti-spine-0.8.7g/spine.h	2010-07-12 12:40:39.663780727 +0300
@@ -135,22 +135,19 @@
 #define SPINE_FORK 0
 
 /* locations to search for the config file */
-#define CONFIG_PATHS 4
-#define CONFIG_PATH_1 ""
-#define CONFIG_PATH_2 "/etc/"
-#define CONFIG_PATH_3 "/etc/cacti/"
-#define CONFIG_PATH_4 "../etc/"
+#define CONFIG_PATHS 1
+#define CONFIG_PATH_1 "/etc/webapps/cacti/"
 
 /* config file defaults */
 #define DEFAULT_CONF_FILE "spine.conf"
 #define DEFAULT_THREADS 5
 #define DEFAULT_DB_HOST "localhost"
 #define DEFAULT_DB_DB "cacti"
-#define DEFAULT_DB_USER "cactiuser"
-#define DEFAULT_DB_PASS "cactiuser"
+#define DEFAULT_DB_USER "mysql"
+#define DEFAULT_DB_PASS ""
 #define DEFAULT_DB_PORT 3306
 #define DEFAULT_DB_PREG 0
-#define DEFAULT_LOGFILE "/var/www/html/cacti/log/cacti.log"
+#define DEFAULT_LOGFILE "/var/log/cacti/cacti.log"
 #define DEFAULT_TIMEOUT 294000000
 
 /* threads constants */
--- cacti-spine-0.8.7a/util.c~	2007-11-26 05:40:21.000000000 +0200
+++ cacti-spine-0.8.7a/util.c	2008-10-05 05:35:03.349764684 +0300
@@ -462,9 +462,6 @@
 	STRNCOPY(set.dbpass, DEFAULT_DB_PASS);
 
 	STRNCOPY(config_paths[0], CONFIG_PATH_1);
-	STRNCOPY(config_paths[1], CONFIG_PATH_2);
-	STRNCOPY(config_paths[2], CONFIG_PATH_3);
-	STRNCOPY(config_paths[3], CONFIG_PATH_4);

 	set.log_destination = LOGDEST_FILE;
 }
--- cacti-spine-1.2.1/spine.conf.dist~	2019-01-20 17:23:29.000000000 +0100
+++ cacti-spine-1.2.1/spine.conf.dist	2019-01-23 14:01:06.618334417 +0100
@@ -39,21 +39,21 @@
 # | RDB_Pass        The remote database password.                           |
 # +-------------------------------------------------------------------------+
 
-DB_Host       localhost
-DB_Database   cacti
-DB_User       cactiuser
-DB_Pass       cactiuser
-DB_Port       3306
+#DB_Host       localhost
+#DB_Database   cacti
+#DB_User       cactiuser
+#DB_Pass       cactiuser
+#DB_Port       3306
 #DB_UseSSL    0
 #DB_SSL_Key
 #DB_SSL_Cert
 #DB_SSL_CA
 
-RDB_Host      localhost
-RDB_Database  cacti
-RDB_User      cactiuser
-RDB_Pass      cactiuser
-RDB_Port      3306
+#RDB_Host      localhost
+#RDB_Database  cacti
+#RDB_User      cactiuser
+#RDB_Pass      cactiuser
+#RDB_Port      3306
 #RDB_UseSSL    0
 #RDB_SSL_Key
 #RDB_SSL_Cert

