--- cacti-spine-0.8.8c/configure.ac~	2014-11-17 06:14:05.000000000 +0000
+++ cacti-spine-0.8.8c/configure.ac	2015-04-06 21:30:05.950921974 +0000
@@ -226,6 +226,8 @@
 fi
 
 for i in $MYSQL_DIR /usr /usr/local /opt /opt/mysql /usr/pkg /usr/local/mysql; do
+  MYSQL_LIB_CHK($i/libx32)
+  MYSQL_LIB_CHK($i/libx32/mysql)
   MYSQL_LIB_CHK($i/lib64)
   MYSQL_LIB_CHK($i/lib64/mysql)
   MYSQL_LIB_CHK($i/lib)
@@ -257,6 +259,8 @@
     test -f $SNMP_DIR/$i/net-snmp-config.h  && SNMP_INCDIR=$SNMP_DIR$i && break
   done
 
+  test -f $SNMP_DIR/libx32/libsnmp.a -o -f $SNMP_DIR/libx32/libsnmp.$ShLib       && SNMP_LIBDIR=$SNMP_DIR/libx32
+  test -f $SNMP_DIR/libx32/libnetsnmp.a -o -f $SNMP_DIR/libx32/libnetsnmp.$ShLib && SNMP_LIBDIR=$SNMP_DIR/libx32
   # Accomodate 64-Bit Libraries
   test -f $SNMP_DIR/lib64/libsnmp.a -o -f $SNMP_DIR/lib64/libsnmp.$ShLib       && SNMP_LIBDIR=$SNMP_DIR/lib64
   test -f $SNMP_DIR/lib64/libnetsnmp.a -o -f $SNMP_DIR/lib64/libnetsnmp.$ShLib && SNMP_LIBDIR=$SNMP_DIR/lib64
@@ -278,6 +282,10 @@
     test -f $i/snmp/include/net-snmp/net-snmp-config.h && SNMP_INCDIR=$i/snmp/include/net-snmp && break
   done
 
+  for i in /usr /usr/local /usr/pkg /usr/snmp /opt /opt/net-snmp /opt/ucd-snmp /opt/snmp /usr/local/snmp; do
+    test -f $i/libx32/libsnmp.a -o -f $i/libx32/libsnmp.$ShLib       && SNMP_LIBDIR=$i/libx32 && break
+    test -f $i/libx32/libnetsnmp.a -o -f $i/libx32/libnetsnmp.$ShLib && SNMP_LIBDIR=$i/libx32 && break
+  done
   # Accomodate 64-Bit Libraries
   for i in /usr /usr/local /usr/pkg /usr/snmp /opt /opt/net-snmp /opt/ucd-snmp /opt/snmp /usr/local/snmp; do
     test -f $i/lib64/libsnmp.a -o -f $i/lib64/libsnmp.$ShLib       && SNMP_LIBDIR=$i/lib64 && break
